uint8 r
uint8 g
uint8 b
uint8 intensity
uint8 index
---
bool result
